ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.ts

```diff
@@ -1364,6 +1432,99 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'UCU Quickmarch Dive Dir',
+      type: 'StartsUsing',
+      netRegex: { id: '26E1', source: 'Bahamut Prime' },
+      condition: (data) => data.trio === 'quickmarch',
+      alertText: (_data, matches, output) => {
+        // Bosses jump, and dive placement is locked once Bahamut starts casting.
+        // Position data is always updated by now, so need to rely on combatant data from OP.
+        // Bahamut will always be on an exact cardinal/intercardinal (w/Nael & Twin on either side)
+        const diveDir = positionTo8DirOutput(parseFloat(matches.x), parseFloat(matches.y));
+        return output.dive!({ dir: output[diveDir]!() });
+      },
+      outputStrings: {
+        dive: {
+          en: 'Diving from ${dir}',
```

Comment:
```suggestion
          en: '${dir} Dive',
```

Just putting the important information first.